integrade/tests/aws_utils.py

```diff
@@ -74,27 +128,30 @@ def create_instances(
     ``create_instances`` waits to return until the instances are all
     running.
 
-    The profile must be named and have its credentials stored in ~/.aws/config
+    The profile must be named and have its credentials stored in
+    $XDG_CONFIG_HOME/integrade/.
     The images are associated with the profiles in
-    ~/.config/integrade/aws_config in the following manner:
+    $XDG_CONFIG_HOME/integrade/aws_image_config.yaml in the following manner:
```

Comment:
is this XDG environment variable set anywhere or do we have to include it? i don't see this var anywhere in the repo being set as well.

Reply:
This is something that is generally set in your environment of your shell. The standard value is ~/.config , but if you WANT it to be something else, you can set it manually.

```
export | grep XDG
declare -x XDG_CURRENT_DESKTOP="GNOME"
declare -x XDG_DATA_DIRS="/home/elijah/.local/share/flatpak/exports/share/:/var/lib/flatpak/exports/share/:/usr/local/share/:/usr/share/"
declare -x XDG_MENU_PREFIX="gnome-"
declare -x XDG_RUNTIME_DIR="/run/user/1000"
declare -x XDG_SEAT="seat0"
declare -x XDG_SESSION_DESKTOP="gnome-xorg"
declare -x XDG_SESSION_ID="2"
declare -x XDG_SESSION_TYPE="x11"
declare -x XDG_VTNR="2"
```